restapi/src/main/java/io/stargate/web/impl/RestApiServer.java

```diff
@@ -170,16 +172,18 @@ protected void configure() {
 
     if (!enableSgv1) {
       logger.info(
-          "Will not register StargateV1 Documents API or RESTv2 endpoints for StargateV2 (System property '{}' {}, enable with 'true')",
+          "Will not register in-Coordinator Documents API or RESTv2 endpoints for Stargate V2 (System property '{}' {}, enable with 'true')",
           SYSPROP_ENABLE_SGV1_REST,
           (enableSgv1RestStr == null)
               ? "UNDEFINED"
               : String.format("set to '%s'", enableSgv1RestStr));
     } else {
-      logger.info(
-          "Registering StargateV1 Documents API and RESTv2 endpoints for StargateV2 (System property '{}' set to '{}')",
+      logger.warn(
+          "Registering in-Coordinator Documents API and RESTv2 API endpoints (System property '{}' set to '{}')",
           SYSPROP_ENABLE_SGV1_REST,
           enableSgv1RestStr);
+      logger.warn(
+          "NOTE: in-Coordinator Documents API and RESTv2 APIs are DEPRECATED in Stargate V2 and scheduled for removal in V3");
```

Comment:
isn't this stated in the readme to be removed in one of the 2.x releases?

Reply:
Thanks. I had an earlier version with this description which is wrong; yes, will be removed.